[Wed Jan 22 15:13:31.726141 2025] [autoindex:error] [pid 9316:tid 2000] [client ::1:52554] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:15:28.062013 2025] [autoindex:error] [pid 9316:tid 2000] [client ::1:52578] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:20:00.375818 2025] [autoindex:error] [pid 9316:tid 1996] [client ::1:52619] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:21:02.671428 2025] [autoindex:error] [pid 9316:tid 1996] [client ::1:52627] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:21:03.328631 2025] [autoindex:error] [pid 9316:tid 1996] [client ::1:52627] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:21:20.779485 2025] [autoindex:error] [pid 4192:tid 1988] [client ::1:52629] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:21:21.306446 2025] [autoindex:error] [pid 4192:tid 1988] [client ::1:52629] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:21:21.490326 2025] [autoindex:error] [pid 4192:tid 1988] [client ::1:52629] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:21:21.652107 2025] [autoindex:error] [pid 4192:tid 1988] [client ::1:52629] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:21:21.797493 2025] [autoindex:error] [pid 4192:tid 1988] [client ::1:52629] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:21:51.339763 2025] [autoindex:error] [pid 4192:tid 1988] [client ::1:52631] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:21:52.477895 2025] [autoindex:error] [pid 4192:tid 1988] [client ::1:52631] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:21:53.046152 2025] [autoindex:error] [pid 4192:tid 1988] [client ::1:52631] AH01276: Cannot serve directory I:/cgi/: No matching DirectoryIndex (index.php,index.pl,index.cgi,index.asp,index.shtml,index.html,index.htm,index.py,default.php,default.pl,default.cgi,default.asp,default.shtml,default.html,default.htm,home.php,home.pl,home.cgi,home.asp,home.shtml,home.html,home.htm) found, and server-generated directory index forbidden by Options directive
[Wed Jan 22 15:59:18.066181 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.066181 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.066181 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.066181 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:           ~~~~~~~~~^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.067184 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 23, in decode\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.067184 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:     return codecs.charmap_decode(input,self.errors,decoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.067184 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.067184 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215: UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 185: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.972001 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.972999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.972999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.972999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:           ~~~~~~~~~^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.972999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 23, in decode\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.972999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:     return codecs.charmap_decode(input,self.errors,decoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.972999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:18.972999 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53152] AH01215: UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 185: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.070734 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.070734 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.070734 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.070734 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:           ~~~~~~~~~^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.070734 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 23, in decode\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.070734 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:     return codecs.charmap_decode(input,self.errors,decoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.070734 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.070734 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215: UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 185: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.738983 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.738983 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.738983 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.738983 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:           ~~~~~~~~~^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.738983 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 23, in decode\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.739988 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:     return codecs.charmap_decode(input,self.errors,decoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.739988 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:33.739988 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215: UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 185: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 15:59:36.366382 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 15:59:36.366382 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 15:59:36.366382 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 15:59:36.366382 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:           ~~~~~~~~~^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:36.366382 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 23, in decode\r: I:/cgi/index.py
[Wed Jan 22 15:59:36.366382 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:     return codecs.charmap_decode(input,self.errors,decoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 15:59:36.366382 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:36.367396 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53153] AH01215: UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 185: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 15:59:48.664997 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 15:59:48.664997 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 15:59:48.664997 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 15:59:48.664997 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:           ~~~~~~~~~^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:48.664997 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 23, in decode\r: I:/cgi/index.py
[Wed Jan 22 15:59:48.664997 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:     return codecs.charmap_decode(input,self.errors,decoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 15:59:48.664997 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:48.664997 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215: UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 185: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 15:59:49.323248 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 15:59:49.323248 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 15:59:49.323248 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 15:59:49.323248 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:           ~~~~~~~~~^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:49.323248 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 23, in decode\r: I:/cgi/index.py
[Wed Jan 22 15:59:49.323248 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:     return codecs.charmap_decode(input,self.errors,decoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 15:59:49.324261 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 15:59:49.324261 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53155] AH01215: UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 185: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 16:00:17.038853 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53160] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 16:00:17.038853 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53160] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 16:00:17.039850 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53160] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 16:00:17.039850 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53160] AH01215:           ~~~~~~~~~^^\r: I:/cgi/index.py
[Wed Jan 22 16:00:17.039850 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53160] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 23, in decode\r: I:/cgi/index.py
[Wed Jan 22 16:00:17.039850 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53160] AH01215:     return codecs.charmap_decode(input,self.errors,decoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 16:00:17.039850 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53160] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:00:17.039850 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53160] AH01215: UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 273: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 16:01:17.390824 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53177] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 16:01:17.390824 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53177] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 16:01:17.390824 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53177] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 16:01:17.390824 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53177] AH01215:           ~~~~~~~~~^^\r: I:/cgi/index.py
[Wed Jan 22 16:01:17.390824 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53177] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 23, in decode\r: I:/cgi/index.py
[Wed Jan 22 16:01:17.390824 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53177] AH01215:     return codecs.charmap_decode(input,self.errors,decoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 16:01:17.390824 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53177] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:01:17.391825 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53177] AH01215: UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 249: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 16:02:46.317938 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 16:02:46.317938 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 16:02:46.318447 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 16:02:46.318447 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:     ~~~~~^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:02:46.318447 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 19, in encode\r: I:/cgi/index.py
[Wed Jan 22 16:02:46.318447 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:     return codecs.charmap_encode(input,self.errors,encoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 16:02:46.318447 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:02:46.318447 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215: UnicodeEncodeError: 'charmap' codec can't encode characters in position 256-264: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 16:02:47.053038 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 16:02:47.053038 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 16:02:47.053038 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 16:02:47.053038 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:     ~~~~~^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:02:47.053038 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 19, in encode\r: I:/cgi/index.py
[Wed Jan 22 16:02:47.053038 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:     return codecs.charmap_encode(input,self.errors,encoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 16:02:47.053038 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:02:47.053038 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53188] AH01215: UnicodeEncodeError: 'charmap' codec can't encode characters in position 256-264: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 16:03:03.419149 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53189] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 16:03:03.419149 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53189] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 16:03:03.419149 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53189] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 16:03:03.419149 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53189] AH01215:     ~~~~~^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:03:03.419149 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53189] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 19, in encode\r: I:/cgi/index.py
[Wed Jan 22 16:03:03.419149 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53189] AH01215:     return codecs.charmap_encode(input,self.errors,encoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 16:03:03.419149 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53189] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:03:03.419149 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:53189] AH01215: UnicodeEncodeError: 'charmap' codec can't encode characters in position 256-264: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 16:03:48.646654 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53191] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 16:03:48.646654 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53191] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 16:03:48.646654 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53191] AH01215:     print(file.read())\r: I:/cgi/index.py
[Wed Jan 22 16:03:48.646654 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53191] AH01215:     ~~~~~^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:03:48.647652 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53191] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 19, in encode\r: I:/cgi/index.py
[Wed Jan 22 16:03:48.647652 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53191] AH01215:     return codecs.charmap_encode(input,self.errors,encoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 16:03:48.647652 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53191] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:03:48.647652 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:53191] AH01215: UnicodeEncodeError: 'charmap' codec can't encode characters in position 256-264: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 16:06:36.930658 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58557] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 16:06:36.930658 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58557] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 16:06:36.930658 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58557] AH01215:     print(file.read(), end="")\r: I:/cgi/index.py
[Wed Jan 22 16:06:36.930658 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58557] AH01215:     ~~~~~^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:06:36.930658 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58557] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 19, in encode\r: I:/cgi/index.py
[Wed Jan 22 16:06:36.931661 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58557] AH01215:     return codecs.charmap_encode(input,self.errors,encoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 16:06:36.931661 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58557] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:06:36.931661 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58557] AH01215: UnicodeEncodeError: 'charmap' codec can't encode characters in position 256-264: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 16:07:53.570608 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58623] AH01215: Traceback (most recent call last):\r: I:/cgi/index.py
[Wed Jan 22 16:07:53.570608 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58623] AH01215:   File "I:\\cgi\\index.py", line 8, in <module>\r: I:/cgi/index.py
[Wed Jan 22 16:07:53.570608 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58623] AH01215:     print(file.read(), end="")\r: I:/cgi/index.py
[Wed Jan 22 16:07:53.570608 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58623] AH01215:     ~~~~~^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:07:53.571606 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58623] AH01215:   File "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.13_3.13.496.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py", line 19, in encode\r: I:/cgi/index.py
[Wed Jan 22 16:07:53.571606 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58623] AH01215:     return codecs.charmap_encode(input,self.errors,encoding_table)[0]\r: I:/cgi/index.py
[Wed Jan 22 16:07:53.571606 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58623] AH01215:            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r: I:/cgi/index.py
[Wed Jan 22 16:07:53.571606 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58623] AH01215: UnicodeEncodeError: 'charmap' codec can't encode characters in position 256-264: character maps to <undefined>\r: I:/cgi/index.py
[Wed Jan 22 16:16:47.117497 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58855] AH02811: script not found or unable to stat: I:/cgi/api1.py, referer: http://cgi213.loc:8080/
[Wed Jan 22 16:20:34.207953 2025] [core:alert] [pid 4192:tid 1960] [client ::1:58888] I:/cgi/.htaccess: RewriteEngine must be On or Off
[Wed Jan 22 16:20:41.149912 2025] [core:alert] [pid 4192:tid 1988] [client ::1:58889] I:/cgi/.htaccess: RewriteEngine must be On or Off
[Wed Jan 22 16:20:44.329015 2025] [core:alert] [pid 4192:tid 1988] [client ::1:58890] I:/cgi/.htaccess: RewriteEngine must be On or Off
[Wed Jan 22 16:20:44.871112 2025] [core:alert] [pid 4192:tid 1960] [client ::1:58891] I:/cgi/.htaccess: RewriteEngine must be On or Off
[Wed Jan 22 16:22:11.237948 2025] [core:alert] [pid 4192:tid 1960] [client ::1:58902] I:/cgi/.htaccess: RewriteEngine must be On or Off
[Wed Jan 22 17:00:20.948226 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:59347] End of script output before headers: api1.py, referer: http://localhost:8080/
[Wed Jan 22 17:00:20.948226 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:59347] AH01215: Traceback (most recent call last):\r: I:/cgi/api1.py, referer: http://localhost:8080/
[Wed Jan 22 17:00:20.948226 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:59347] AH01215:   File "I:\\cgi\\api1.py", line 17, in <module>\r: I:/cgi/api1.py, referer: http://localhost:8080/
[Wed Jan 22 17:00:20.948226 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:59347] AH01215:     query_string = urllib.parse.unquote(envs['QUERY_STRING'], encoding="utf-8")\r: I:/cgi/api1.py, referer: http://localhost:8080/
[Wed Jan 22 17:00:20.948226 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:59347] AH01215:                    ^^^^^^\r: I:/cgi/api1.py, referer: http://localhost:8080/
[Wed Jan 22 17:00:20.948226 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:59347] AH01215: NameError: name 'urllib' is not defined. Did you forget to import 'urllib'?\r: I:/cgi/api1.py, referer: http://localhost:8080/
[Wed Jan 22 17:42:49.934394 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58125] AH02811: script not found or unable to stat: I:/New folder (7)/cgi-bin/api, referer: http://localhost:8080/
[Wed Jan 22 17:42:51.508012 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58125] AH02811: script not found or unable to stat: I:/New folder (7)/cgi-bin/api1.py, referer: http://localhost:8080/
[Wed Jan 22 17:42:52.336435 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58125] AH02811: script not found or unable to stat: I:/New folder (7)/cgi-bin/api1.py, referer: http://localhost:8080/
[Wed Jan 22 17:42:52.775407 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58125] AH02811: script not found or unable to stat: I:/New folder (7)/cgi-bin/api, referer: http://localhost:8080/
[Wed Jan 22 17:42:55.354239 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58125] AH02811: script not found or unable to stat: I:/New folder (7)/cgi-bin/api1.py, referer: http://localhost:8080/
[Wed Jan 22 17:43:01.350461 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:58126] AH02811: script not found or unable to stat: I:/New folder (7)/cgi-bin/api, referer: http://localhost:8080/
[Wed Jan 22 17:58:21.054359 2025] [cgi:error] [pid 4192:tid 1960] [client ::1:58281] Premature end of script headers: api1.py, referer: http://localhost:8080/
[Wed Jan 22 17:58:22.108443 2025] [cgi:error] [pid 4192:tid 1988] [client ::1:58282] Premature end of script headers: api1.py, referer: http://localhost:8080/
